queries.js

```diff
@@ -0,0 +1,63 @@
+const Pool = require('pg').Pool
+const pool = new Pool({
+  user: 'admin',
+  host: 'localhost',
+  database: 'api',
+  password: 'admin_pass',
+  port: 5432,
+})
+
+const getNotes = (request, response) => {
+    pool.query('SELECT * FROM notes ORDER BY id ASC', (error, results) => {
+        if (error) {
+            throw error
+        }
+        response.status(200).json(results.rows)
+    })
+}
+
+const getNoteByID = (request, response) => {
+    const id = parseInt(request.params.id)
+
+    pool.query('SELECT * FROM notes WHERE id = $1', [id], (error, results) => {
```

Comment:
I think you should be selecting the column instead of the whole db items

Reply:
ok